src/test/kotlin/org/jetBrains/research/tasktracker/tracking/Tasks.kt

```diff
@@ -0,0 +1,136 @@
+package org.jetBrains.research.tasktracker.tracking
+
+import org.jetBrains.research.tasktracker.config.MainTaskTrackerConfig
+import org.jetBrains.research.tasktracker.models.Extension
+import org.jetBrains.research.tasktracker.tracking.task.Task
+
+val task1 = object : Task {
+    override fun getContent(): String {
+        return "int main(){return 0;}"
+    }
+
+    override fun getRelativeFilePath(): String? {
+        return null
+    }
+
+    override fun getFileName(): String {
+        return "task1"
+    }
+
+    override fun getExtension(): Extension {
+        return Extension.CPP
+    }
+}
+
+val task2 = object : Task {
+    override fun getContent(): String? {
+        return null
+    }
+
+    override fun getRelativeFilePath(): String {
+        return "tasks"
+    }
+
+    override fun getFileName(): String {
+        return "task2"
+    }
+
+    override fun getExtension(): Extension {
+        return Extension.KOTLIN
+    }
+}
+
+val task3 = object : Task {
+    override fun getContent(): String? {
+        return null
+    }
+
+    override fun getRelativeFilePath(): String? {
+        return null
+    }
+
+    override fun getFileName(): String {
+        return "task3"
+    }
+
+    override fun getExtension(): Extension {
+        return Extension.JAVA
+    }
+}
+
+val task4 = object : Task {
+    override fun getContent(): String? {
+        return null
+    }
+
+    override fun getRelativeFilePath(): String {
+        return "tasks"
+    }
+
+    override fun getFileName(): String {
+        return "task4"
+    }
+
+    override fun getExtension(): Extension {
+        return Extension.PYTHON
+    }
+}
+
+val task5 = object : Task {
+    override fun getContent(): String? {
+        return null
+    }
+
+    override fun getRelativeFilePath(): String? {
+        return null
+    }
+
+    override fun getFileName(): String {
+        return "task5"
+    }
+
+    override fun getExtension(): Extension {
+        return Extension.JUPYTER
+    }
+}
+
+val CPP_CONTENT =
```

Comment:
I'd suggest files with content in the resource folder and build mock tasks by these files. In the getDefaultContentTest get content from a file by task (getFileName + getExtension) for the actual content. What do you think?

Reply:
I think this may not work for all file extensions. For Kotlin and Java files, it is important for us to know the package, which, in turn, depends on the plugin's name.

Reply:
but you can change test files if the plugin name is changed.... I don't expect that we will change the plugin name often